src/workloads/execution/BackgroundTTLDeletions.yml

```diff
@@ -0,0 +1,187 @@
+CollectionCount: &NumColls 1
+
+Document: &generatedDoc
+  expireDate: {^Now: {}}  # Instantly expire the document
+  y: &y {^RandomInt: {min: -1000, max: 1000}}
+  b: {^RandomString: {length: 15}}
+
+ActorTemplates:
+- TemplateName: WritesActorTemplate
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: WritesActor}}
+    Type: CrudActor
+    Database: &db test
+    Threads: {^Parameter: {Name: "Threads", Default: 1}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: {^Parameter: {Name: "ActivePhases", Default: [1]}}
+        NopInPhasesUpTo: *NumPhases
+        PhaseConfig:
+          Duration: 3 minutes
+          CollectionCount: *NumColls
+          Operations:
+          - OperationName: insertMany
+            OperationCommand:
+              Documents:  # Insert 50 documents at a time
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+
+
+Actors:
+- Name: Setup
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *NumPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: admin
+        Operations:
+        - OperationName: AdminCommand
+          OperationCommand:
+            setParameter: 1
+            ttlMonitorSleepSecs: 1
+        - OperationName: AdminCommand
+          OperationCommand:
+            setParameter: 1
+            storageEngineConcurrentWriteTransactions: 32
+        SleepAfter: 60 seconds  # Wait 60 seconds so that the new ttl sleep config is active.
```

Comment:
Would a quiesce stage be sufficient? A minute seems generous. Alternatively is there something we can check?

Reply:
I'm not sure the TTL config change would get picked up by the quiesce stage. Internally the TTL performs a [sleep](https://github.com/mongodb/mongo/blob/c0953d92e0a07dd130a0f4a4ac26a9fc4d7ef7f7/src/mongo/db/ttl.cpp#L271) for the ttl sleep config amount of seconds, so it will initially sleep for 60 seconds (default value) before changing to the new config.

Reply:
As an FYI, I've created [SERVER-70055](https://jira.mongodb.org/browse/SERVER-70055) to fix this eventually.

Reply:
Thanks.